tests/core/test_hookenv.py

```diff
@@ -1288,7 +1288,7 @@ def test_relation_set_file(self, check_call, check_output, remove):
         self.assertEqual("--file", command[1])
         temp_file = command[2]
         with open(temp_file, "r") as f:
-            self.assertEqual("{foo: bar}", f.read().strip())
+            self.assertEqual("foo: bar", f.read().strip())
```

Comment:
Can you add a pointer to why this has changed and is it backward compatibile with earlier py3's and py2.7?

Reply:
This is due to pyyaml 5.3 rather than 3.10  and since test-requirements.txt is pinned to 5.3 these changes in tests should be good.

Reply:
yes, this is because testing now takes place using pyyaml 5.3  Python 3.8 was thoroughly unhappy with pyyaml 3.10.  You might see on travis that there were errors building that version of pyyaml.